### PR TITLE
Add ability to customize order of output for edge tpu detector models

### PR DESCRIPTION
**Pull request by alec-groff**
Add order to config.model to allow the user the ability to specify the order of boxes, scores, class_ids, and count for their edge tpu detector output. The default value is the current setting.

### CODE REVIEW

File: frigate/detectors/plugins/edgetpu_tfl.py [hunk 1]
```diff
@@ -44,6 +44,7 @@ def __init__(self, detector_config: EdgeTpuDetectorConfig):
                 model_path=detector_config.model.path,
                 experimental_delegates=[edge_tpu_delegate],
             )
+            self.order = detector_config.model.order
```
Comment (NickM-27) [inline frigate/detectors/plugins/edgetpu_tfl.py:47]: I don't think there is a reason to set `self.order` every time, just set it once in init

File: frigate/detectors/detector_config.py [hunk 2]
```diff
@@ -51,6 +51,9 @@ class ModelConfig(BaseModel):
     model_type: ModelTypeEnum = Field(
         default=ModelTypeEnum.ssd, title="Object Detection Model Type"
     )
+    order: list[int] = Field(
```
Comment (NickM-27) [inline frigate/detectors/detector_config.py:54]: this config would apply to every detector so it needs to be used for every detector or perhaps the default value should be `None` and the field changes to be called `custom_model_output_order` or something

Comment (NickM-27) [inline frigate/detectors/detector_config.py:54]: I think the name should be more descriptive like `model_output_tensor_order` or something

**Review by NickM-27 — CHANGES_REQUESTED** [review]

**Comment by bazylhorsey** [comment]
Any update on this?

**Pull request closed without merging**